types/v56/types.go

```diff
@@ -256,7 +256,7 @@ type VAppNetwork struct {
 	Type     string `xml:"type,attr,omitempty"`
 	ID       string `xml:"id,attr,omitempty"`
 	Name     string `xml:"name,attr"`
-	Deployed bool   `xml:"deployed,attr"`
+	Deployed *bool  `xml:"deployed,attr"`
```

Comment:
I know it is not added now, but reviewing the PR struck me - what does being `deployed` mean in this case? Is it attached to vApp (although all vApp networks should be attached in the end)? 
Maybe you could add a line to comment

Reply:
add comment from API